### ISSUE TITLE
Save Entry Event not getting triggered properly

### ISSUE THREAD

**Issue by BenParizek** [barrelstrength/craft-sprout-email#164]
Not sure if these errors are relevant but they were logged while the email did not successfully send even though the settings appeared to meet all these conditions.

```
[error][barrelstrength\sproutbaseemail\services\NotificationEmailEvents::handleDynamicEvent] When an entry is saved event does not validate: {"event":["When an Entry is saved Event does not match any scenarios.","\"When an entry is created\" is selected but the entry is being updated."]}

[error][barrelstrength\sproutbaseemail\services\NotificationEmailEvents::handleDynamicEvent] When an entry is saved event does not validate: {"event":["The `EntriesSave` Notification Event only triggers when an Entry is saved in a live scenario."]}

[error][barrelstrength\sproutbaseemail\services\NotificationEmailEvents::handleDynamicEvent] When an entry is saved event does not validate: {"event":["The `EntriesSave` Notification Event only triggers when an Entry is saved in a live scenario."]}
```

Related:
- https://github.com/craftcms/cms/issues/6130

**Comment by BenParizek** [comment]
With the introduction of Drafts, Craft changed the way in which it identifies new Entries. This logic will need to be updated.

**Comment by pixleight** [comment]
I just ran into this problem on a site and found this issue; checking to see if it's still in development or due to be included in a release soon.

**Comment by joshuabaker** [comment]
@BenParizek Is there any update on this? 

It’s been a while since this issue was updated and I note that the `bugfix/new-notifications` branch isn’t solving the issue in our Craft 3.5.15.1 install.

**Comment by ccchapman** [comment]
I am experiencing an issue after upgrading that may or may not be relevant here. (Please let me know if it sounds unique.) Most of our Sprout Email notifications are working as expected, but not one that relies on the `barrelstrength\sproutemail\events\notificationevents\UsersSave` event. I have the `When user is created?` setting selected along with one user group.

Interestingly, the notification does fire as expected if I create a user through the CP. It does not work using Craft's first class `users/save-user` action.

It sounds like there may be some ongoing work re: new element event notifications, so I wanted to check if that's the case and my issue is relevant.

```
Sprout Base 6.0.2
Sprout Email 4.4.8
Sprout Encode Email 2.0.7
Craft CMS 3.5.15.1
```

**Comment by BenParizek** [comment]
For the record, this has been a bit of a moving target between Craft 3.2 and Craft 3.75. I believe things will be more stable moving forward. I've summarized the history of the changes in Craft in [this post on Stack Exchange](https://craftcms.stackexchange.com/a/38169/115).

And I've just released Sprout Email v4.4.10 which gets things in line with the latest save entry behavior.
